docs/UserGuide.md

```diff
@@ -38,11 +43,51 @@ It is optimized for fast typists and replicates a Command Line Interface (CLI) s
 
    * **`exit`** : Exits the app.
 
-6. Refer to the [things to note](#3-things-to-note) below for details of each command.
+6. You can refer to the [things to note](#3-things-to-note) below for details of each command.
```

Comment:
description doesnt really match the actual content? (things to note is misc. info about commands, not details of each command)
maybe something like this instead?
```suggestion
6. You can jump straight to [here](#6-what-you-can-do) for a summary of the commands.
```
honestly can also remove it, not sure whats the use case for the hyperlink here